coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -36,10 +37,31 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
         """
         Add ignore comment
         """
-        comment_delimiter = Language[
-            language].get_default_version().comment_delimiter
-        ignore_comment = (str(comment_delimiter) + ' Ignore ' + result.origin +
-                          '\n')
+        try:
+            singleline_comment_delimiter = Language[
+                language].get_default_version().comment_delimiter
+            ignore_comment = (str(singleline_comment_delimiter) + ' Ignore ' +
+                              result.origin + '\n')
+        except AttributeError:
+            # singleline comments not supported by language
+            try:
+                multiline_comment_delimiter = Language[
+                    language].get_default_version().multiline_comment_delimiters
+                start_multiline_comment = list(
+                                        multiline_comment_delimiter.keys())[0]
+                end_multiline_comment = list(
+                                        multiline_comment_delimiter.values())[0]
+                ignore_comment = (str(start_multiline_comment) + ' Ignore ' +
+                                  result.origin + ' ' +
+                                  str(end_multiline_comment) + '\n')
+            except AttributeError:
+                # multiline comments also not supported by language
+                logging.warning(
+                    'coala does not support Ignore in "{language}". Consider'
+                    ' opening an issue at https://github.com/coala/coala/issues'
+                    ' so we can add support for this language.'.format(
+                        language=language))
+                return file_diff_dict
```

Comment:
wether or not you're around has usually no influence on reviewers, we're trying to do full reviews wen we can but if we can't, well... we can't ;)